src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -521,28 +524,66 @@ const SearchFieldList = ({ onSearch }: Props) => {
                       select
                     />
                   ) : (
-                    <Grid item width={225}>
-                      <Slider
-                        getAriaLabel={() => 'graduationYearRange'}
-                        value={graduationYearRange}
-                        onChange={handleSliderChange}
-                        valueLabelDisplay="auto"
-                        getAriaValueText={toString}
-                        min={1889}
-                        max={currentYear}
-                        disabled={!searchParams.includeAlumni}
-                        color="secondary"
-                      />
-                      <Typography fontSize={15} align="center">
-                        {graduationYearRange[0]}-{graduationYearRange[1]}
-                      </Typography>
+                    <Grid item container spacing={1}>
+                      <Grid item xs={6} md={6.75}>
+                        <SearchField
+                          name="initial_year"
+                          value={searchParams.initial_year}
+                          updateValue={handleUpdate}
+                          options={Array.from(
+                            {
+                              length:
+                                Number(searchParams.final_year) > 0
+                                  ? Number(searchParams.final_year) - 1889 + 1
+                                  : currentYear - 1889 + 1,
+                            },
+                            (_, i) => ({
+                              value:
+                                Number(searchParams.final_year) > 0
+                                  ? (Number(searchParams.final_year) - i).toString()
+                                  : (currentYear - i).toString(),
```

Comment:
```suggestion
                                  ? `${searchParams.final_year - i}`
                                  : `${currentYear - i}`
```

Personal preference for how code looks. However, I think @EjPlatzer should have a final say on this. I like to just accept that javascript is cursed and thus use the fact that the typeless string numbers can be read by the compiler as integers during any form of subtraction. However, I can see how this could be confusing for code reviewers. 

Also I've always preferred using formatted strings over using casting functions. Once again, personal preference for code prettiness